docs/design/README.md

```diff
@@ -30,9 +30,15 @@ Writing a design document can promote us to think deliberately and gather knowle
 - [Proposal: A SQL Planner based on the Volcano/Cascades model](./2018-08-29-new-planner.md)
 - [Proposal: Implement Radix Hash Join](./2018-09-21-radix-hashjoin.md)
 - [Proposal: Maintaining histograms in plan](./2018-09-04-histograms-in-plan.md)
+- [Proposal: TiDB DDL architecture](./2018-10-08-online-DDL.md)
+- [Proposal: Table Partition](./2018-10-19-table-partition.md)
```

Comment:
I think they are both finished, @zimulala @tiancaiamao please confirm

Reply:
Yes, table partition in the proposal are finished... 
Finished ones are not put here? @zz-jason

Reply:
Yeah, finished proposals should be put in the `Completed` section I think.